client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -969,7 +969,8 @@ export class WorkspaceView extends React.Component<any, any> {
                         {!(this.state.isAuthenticated && !isActive) &&
                           workspace.message && (
                             <Alert style={{ border: "1px solid #ddd" }}>
-                              {workspace.childWorkspaces &&
+                              {hasParent &&
```

Comment:
This is a check to ensure a rare edge case warning only gets displayed in non-root workspaces. Put into this PR for convenience (issue was noticed while testing this PR) but not related to scheduler modifications. Same for email change a few lines below.